src/index.html

```diff
@@ -4,10 +4,276 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link
+      rel="icon"
+      href="./images/favicon.png"
+      type="image/x-icon"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&family=Space+Grotesk:wght@400;500;700&family=Space+Mono&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="header">
+      <div class="nav header__nav">
+        <a href="#" class="nav__item">
+          <img
+            src="./images/logo-transparent.png" alt="NOTHING logo"
+            class="nav__logo"
+          >
+        </a>
+        <div class="nav__icons">
+          <a class="nav__item phone" href="tel:+1 234 555-55-55">
```

Comment:
it's not possible to click the phone number when hover, it shouldn't disappear

Reply:
Shouldn't the icon be clickable instead of phone number?